### PR TITLE
[#1177] Show new plan contact count in menu

### PR DESCRIPTION
**Pull request by Bartvaderkin**

### CODE REVIEW

File: src/open_inwoner/plans/migrations/0013_auto_20230223_1115.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
+        ("accounts", "0055_user_image"),
```
Comment (Bartvaderkin) [inline src/open_inwoner/plans/migrations/0013_auto_20230223_1115.py:9]: I had to edit this existing migration to get the migration test to work.

**Review by vaszig — COMMENTED** [review]
One remark concerning the creator of the plan. Do we want to show the count to the creator as well?I would expect this only for the contact, since the creator is already aware of the plan at this point.

File: src/open_inwoner/accounts/models.py [hunk 2]
```diff
@@ -282,6 +283,12 @@ def is_email_of_contact(self, email):
             or self.contacts_for_approval.filter(email=email).exists()
         )
 
+    def get_plan_contact_new_count(self):
```
Comment (vaszig) [inline src/open_inwoner/accounts/models.py:286]: Would it be better if we put these in the `query.py`?

Comment (Bartvaderkin) [reply src/open_inwoner/accounts/models.py:286]: query.py is usually for things about collections of the model, while this is instance specific so can be an instance method

**Review by vaszig — CHANGES_REQUESTED** [review]
As mentioned in the previous comment, the number is shown for the creator as well.

**Comment by Bartvaderkin** [comment]
@vaszig Review is mostly about last two commits, although I had to rebase and force push.

**Review by vaszig — APPROVED** [review]

**Comment by alextreme** [comment]
Can be merged

**Pull request merged**